packages/strapi-generate-model/templates/bookshelf/model.settings.template

```diff
@@ -1,6 +1,5 @@
 {
   "kind": "collectionType",
-  "connection": "<%= connection %>",
```

Comment:
Might need a refactor so it is set only if the connection arg is passed in the CLI

Reply:
632432bbc5342ae7c54834fa68c37a23aa8412e6